frontend/src/pages/mentoringCreate/components/MentoringCreateForm/MentoringCreateForm.tsx

```diff
@@ -36,6 +39,12 @@ function MentoringCreateForm() {
     [],
   );
 
+  const { priceErrorMessage } = usePriceError(mentoringData.price);
+  const { introduceErrorMessage } = useIntroduceError(
+    mentoringData.introduction,
+  );
+  const { careerErrorMessage } = useCareerError(mentoringData.career);
+
```

Comment:
p2: 혹시 훅을 제거하고 에러 메시지를 파생 상태로 만드는 건 어떨까? 🤔

이미 검증하는 함수를 잘 만들어놔서 상태 값이 바뀔 때마다 검증해주는 역할은 useEffect를 쓰지 않아도 파생상태로 충분히 될 거 같아 👍

```suggestion
const priceErrorMessage = priceValidator(mentoringData.price);
  const introduceErrorMessage = introduceValidator(mentoringData.introduction);
  const careerErrorMessage = careerValidator(mentoringData.career);

```

Reply:
그러네 어처피 값 변경 감지는 handleMentoringDataChange 해주기때문에 useEffect로 값 하나하나 변경상태를 추적하지 않아도 된다는 뜻인거지?

Reply:
[refactor: 불필요한 훅 및 타입 과 검증 제거](https://github.com/woowacourse-teams/2025-Fit-toring/pull/307/commits/680523ad9b60e6689f8530cd9d9cd7874e1f341e)